db/seeds/users.rb

```diff
@@ -1,188 +1,188 @@
+puts User.create(first: 'Souffle', last: 'Cookie' , email: 'SouffleCookie@Cookie.com', user_name: 'SouffleCookie', street: '70575 Main St.' , city: 'Miami', state: 'FL', zip: 66666, password: 'password', password_confirmation: 'password', admin: false).email
+puts User.create(first: 'Croissant', last: 'Cookie' , email: 'CroissantCookie@Cookie.com', user_name: 'CroissantCookie', street: '85245 Main St.' , city: 'Topeka', state: 'KS', zip: 94625, password: 'password', password_confirmation: 'password', admin: false).email
+puts User.create(first: 'Pie', last: 'Cotton' , email: 'PieCotton@Cotton.com', user_name: 'PieCotton', street: '71036 Main St.' , city: 'Fairfax', state: 'VA', zip: 22033, password: 'password', password_confirmation: 'password', admin: false).email
+puts User.create(first: 'Sugar', last: 'Cotton' , email: 'SugarCotton@Cotton.com', user_name: 'SugarCotton', street: '96502 Main St.' , city: 'Topeka', state: 'KS', zip: 94625, password: 'password', password_confirmation: 'password', admin: false).email
+puts User.create(first: 'Plum', last: 'Cream' , email: 'PlumCream@Cream.com', user_name: 'PlumCream', street: '77005 Main St.' , city: 'Newtown', state: 'CT', zip: 16474, password: 'password', password_confirmation: 'password', admin: false).email
+puts User.create(first: 'Wafer', last: 'Cream' , email: 'WaferCream@Cream.com', user_name: 'WaferCream', street: '96164 Main St.' , city: 'Newtown', state: 'CT', zip: 16474, password: 'password', password_confirmation: 'password', admin: false).email
+puts User.create(first: 'Gummies', last: 'Cream' , email: 'GummiesCream@Cream.com', user_name: 'GummiesCream', street: '35489 Main St.' , city: 'Fairfax', state: 'VA', zip: 22033, password: 'password', password_confirmation: 'password', admin: false).email
+puts User.create(first: 'Jelly', last: 'Cream' , email: 'JellyCream@Cream.com', user_name: 'JellyCream', street: '10415 Main St.' , city: 'Springfield', state: 'MO', zip: 57294, password: 'password', password_confirmation: 'password', admin: false).email
+puts User.create(first: 'Beans', last: 'Croissant' , email: 'BeansCroissant@Croissant.com', user_name: 'BeansCroissant', street: '38634 Main St.' , city: 'Springfield', state: 'MO', zip: 57294, password: 'password', password_confirmation: 'password', admin: false).email
+puts User.create(first: 'Cake', last: 'Croissant' , email: 'CakeCroissant@Croissant.com', user_name: 'CakeCroissant', street: '76930 Main St.' , city: 'Fairfax', state: 'VA', zip: 22033, password: 'password', password_confirmation: 'password', admin: false).email
+puts User.create(first: 'Icing', last: 'Croissant' , email: 'IcingCroissant@Croissant.com', user_name: 'IcingCroissant', street: '51733 Main St.' , city: 'Denver', state: 'CO', zip: 80202, password: 'password', password_confirmation: 'password', admin: false).email
+puts User.create(first: 'Tiramisu', last: 'Croissant' , email: 'TiramisuCroissant@Croissant.com', user_name: 'TiramisuCroissant', street: '60
```

Comment:
Line is too long. [266/80]<br>Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not write to stdout. Use Rails' logger if you want to log.